docs/changehistory/NextVersion.md

```diff
@@ -50,3 +51,38 @@ The Xml and JSON representations of a custom attribute (and related TypeScript i
 ## Improve the performance of the ECSchemaRpcLocater
 
 Improve the performance of the ECSchemaRpcLocater by making all of the underlying ECSchemaRpcInterface methods GET by default so responses are cached by default. Previously each client had to set the methods to be GET or they would default to POST and were not cached.
+
+## Mathematical operation parsing
+
+The quantity formatter now supports parsing mathematical operations. Ex :
+```Typescript
+// Simplified example :
+// "5 ft + 12 in + 1 yd -1 ft 6 in" => 7.5 (feet)
+// (5 + 1 + 3 - 1.5) => 7.5
+
+// Asynchronous implementation
+const quantityProps = await Parser.parseIntoQuantity("5 ft + 12 in + 1 yd -1 ft 6 in", format, unitsProvider);
+quantityProps.magnitude // 7.5 (feet)
+
+// Synchronous implementation
+const parseResult = Parser.parseToQuantityValue("5 ft + 12 in + 1 yd -1 ft 6 in", format, feetConversionSpecs);
+parseResult.value // 7.5 (feet)
+```
+
+### Limitations
+Only plus(`+`) and minus(`-`) signs are supported for now.
+Other opertaors will end up returning a parsing error or an invalid input result.
+
+The parsing of mathematical operations is disabled by default.
+To enable it, you can override the default QuantityFormatter. Ex :
+```Typescript
+  // App specific
+  const quantityType = QuantityType.LengthEngineering;
+
+  // Default props for the desired quantityType
+  const props = IModelApp.quantityFormatter.getFormatPropsByQuantityType(quantityType);
+
+  // Override the formatter and enable mathematical operations.
+  await IModelApp.quantityFormatter.setOverrideFormat(quantityType, { ...props, allowMathematicOperations: true });
+```
+
```

Comment:
While we should document what has been added in NextVersion.md detailed docs like these are lost if only put in NextVersion.md.  I suggest adding these docs to some more permanent place, putting a simple summary here and linking to the full docs

Reply:
Will be resolved outside of this PR